src/benchmarklib/benchmark_sql_executor.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "sql/sql_pipeline.hpp"
+#include "utils/sqlite_wrapper.hpp"
+
+namespace opossum {
+
+class TransactionContext;
+
+// This class provides SQL functionality to BenchmarkItemRunners. See AbstractBenchmarkItemRunner:_execute_item.
```

Comment:
+ "every query executed via this interface will also, if requested, be validated against sql and/or visualized"

Reply:
+ "represents one transaction"?

Reply:
>     * "every query executed via this interface will also, if requested, be validated against sql and/or visualized"

Done, even though I don't see a significant difference to the previous comment on the constructor.

>     * "represents one transaction"?

"For now, we expect items to use a single transaction, which is why the BenchmarkSQLExecutor executes all queries in the same context."

Reply:
(I first read class comments, not constructor comments.)